app/src/docs/scripts/search/indexer.js

```diff
@@ -0,0 +1,52 @@
+import {
```

Comment:
Any thought on moving contents of this file to `bin/indexer` & `chmod +x bin/indexer`. A couple of other changes would have to happen, i.e.:

```js
#!/usr/bin/env node -r esm

const path = require('path')

const storePath = path.resolve(__dirname, '../src/docs/components/Search/index/store.json')

const indexPath = path.resolve(__dirname, '../src/docs/components/Search/index/index.json')

// The rest of the script here. Above paths can be used to point to store and index jsons.
```

To run the indexing you'd then execute `bin/indexer` or `DATA_UNIONS_DOCS=on bin/indexer`.

Reply:
I was thinking `app/scripts` like Juha mentioned would be a good home for this, then bind it to an npm run script. Does putting it in `bin/indexer` have any benefits over that approach @mondoreale ?

Reply:
@fonty1, benefits not really. You interact with it differently, naturally.

Script in package would look like this
```json
"indexer": "node -r esm /path/to/indexer.js",
```
Then you run it like this
```
npm run indexer
```
With `bin/indexer` you don't need a script cause you can run it directly. If you want a trace of it in package then you can also define a script
```json
"indexer": "bin/indexer",
```

At the end of the day it's what you like better.

Reply:
hmm, I'll keep it where it is for now, since it's there. If there's issues I'll throw it in the `bin` :)